<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uyeler</class>
 <widget class="QDialog" name="uyeler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>uye ad</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_uye_no">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>160</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_uye_ad">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>uye soyad</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_uye_soyad">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sil">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>450</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>sil</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tüm Üyeler</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_uyeler">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>321</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ÜYE İŞLEMLERİ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_yeni_kayit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>yeni kayıt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guncelle">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>güncelle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>uye no</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
